gazebo_ros/package.xml

```diff
@@ -29,6 +29,8 @@
   <depend>std_srvs</depend>
   <depend>tinyxml_vendor</depend>
 
+  <exec_depend>launch_ros</exec_depend>
```

Comment:
Just noting this fix is unrelated to the PR. It looks like it's needed here: https://github.com/ros-simulation/gazebo_ros_pkgs/blob/44427932baca86fb78c06efe41f2c6311d088b04/gazebo_ros/launch/spawn_entity_demo.launch.py#L25